chia/util/condition_tools.py

```diff
@@ -11,19 +11,42 @@
 from chia.util.clvm import int_from_bytes
 from chia.util.errors import ConsensusError, Err
 from chia.util.ints import uint64
+from clvm import SExp
 
 # TODO: review each `assert` and consider replacing with explicit checks
 #       since asserts can be stripped with python `-OO` flag
 
 
+def as_atom_list(obj: SExp) -> List[bytes]:
```

Comment:
condition_tools.py makes sense, but it's a little odd to move it away from program.py